src/app/learn/reference/configuration/index.html

```diff
@@ -258,6 +258,11 @@
         </li>
       </ul>
     </li>
+    <li>
+      <p>
+        <stache-code>pipelineSettings</stache-code> — Specifies configuration options for build and release pipelines. The data type of this property is <stache-code>any</stache-code>. This property is for internal Blackbaud use only.
```

Comment:
@Blackbaud-BobbyEarl, I added a couple sentences after talking with @Blackbaud-SteveBrush about this property. Please let me know what you think. Thanks.

Reply:
What do you think about the following?

```
Specifies configuration options for build and release pipelines. The data type of this property is <stache-code>any</stache-code>. Users are free to use this property as they see fit to control their build process.  Internally, Blackbaud is using it in such a manner.
```

Even writing that I'm not a huge fan.  I'm trying to convey that it's not necessarily a Blackbaud only property, meaning anyone can use it for whatever they want (hence the `any` type), but that Blackbaud is using it internally.

Reply:
What do you think of replacing my last sentence and your last 2 sentences with: "SKY UX does not provide public access to a pipeline, but Blackbaud uses this property internally to control our build processes. External consumers can use this property to define their own pipeline values for their build machines, but SKY UX not support those build processes beyond providing this container for custom values."